src/lib/y2ntp_client/widgets/pool_widgets.rb

```diff
@@ -80,9 +86,11 @@ def store
       end
 
       def help
-        _("<b>Quick Initial Sync</b> specifies whether the 'iburst' option is used. This option " \
-        "sends 4 poll requests in 2 second intervals during the initialization. It is useful for " \
-        "a quick synchronization during the start of the machine.")
+        "<p>" +
+          _("<b>Quick Initial Sync</b> specifies whether the 'iburst' option is used. This " \
```

Comment:
NP: wrong indentation?

Reply:
Another thing: why in some places (see below, line 121) the `<p>` tag is inside the translation and here is outside?

Reply:
Well, in this case, the text was already translated, so I avoided to modify the translation but only added the paragraph to me more legible, although maybe is true that I was careful in some case but no in others.

Reply:
Make sense. Thank you for the explanation :)